tests/foreman/cli/test_provisioning.py

```diff
@@ -15,15 +15,424 @@
 :Upstream: No
 """
 
+import logging
+import math
+import paramiko
+import pytest
+import time
+
+from nailgun import entities
+from fauxfactory import gen_string
+from robottelo.api.utils import (
+    call_entity_method_with_timeout,
+    enable_rhrepo_and_fetchid,
+    promote,
+    upload_manifest,
+)
+from robottelo.config import settings
+from robottelo.constants import (
+    DEFAULT_PTABLE,
+    DEFAULT_SUBSCRIPTION_NAME,
+    PRDS,
+    REPOS,
+    REPOSET,
+)
+from robottelo.cli.host import (
+    Host,
+)
 from robottelo.decorators import (
-    stubbed,
     tier3,
 )
+from robottelo.helpers import (
+    get_nailgun_config,
+)
+from robottelo import manifests
+
+LOGGER = logging.getLogger('robottelo')
+
+
+@pytest.fixture(scope='session')
+def user_credentials():
+    # Create a new user with admin permissions
+    login = gen_string('alphanumeric')
```

Comment:
good to define range of 255 so that we don't have random failures

Reply:
Done, thank you! Also changed to "alpha".